### PR TITLE
Upgrade to airbase 50

### PR DESCRIPTION
**Pull request by cberner**

### CODE REVIEW

**Comment by electrum** [comment]
Update commit message to indicate this upgrades Jetty and add releases notes. Otherwise looks good.

File: http-client/src/test/java/io/airlift/http/client/jetty/TestJettyHttpsClient.java [hunk 1]
```diff
@@ -75,7 +76,7 @@ public void tearDownHttpClient()
         }
     }
 
-    @Test(expectedExceptions = {SSLHandshakeException.class, EOFException.class})
+    @Test(expectedExceptions = {SSLHandshakeException.class, EOFException.class, ClosedChannelException.class})
```
Comment (electrum) [inline http-client/src/test/java/io/airlift/http/client/jetty/TestJettyHttpsClient.java]: Maybe this should just be `IOException`?

Comment (johngmyers) [inline http-client/src/test/java/io/airlift/http/client/jetty/TestJettyHttpsClient.java]: eclipse/jetty.project#444

Comment (cberner) [inline http-client/src/test/java/io/airlift/http/client/jetty/TestJettyHttpsClient.java]: @johngmyers ya, I asked the Jetty folks about that. They said we shouldn't rely on the exception type in this case

Comment (johngmyers) [inline http-client/src/test/java/io/airlift/http/client/jetty/TestJettyHttpsClient.java]: Well, the exception they're throwing now is entirely inappropriate and misleading. It's unlikely anyone would be able to diagnose the condition from that. But that's a bug in Jetty.

**Pull request merged**